mne/decoding/tests/test_time_gen.py

```diff
@@ -46,10 +46,19 @@ def test_generalization_across_time():
     """Test time generalization decoding
     """
     from sklearn.svm import SVC
-    from sklearn.linear_model import RANSACRegressor, LinearRegression
+    # KernelRidge is used for testing 1) regression analyses 2) n-dimensional
+    # predictions.
+    from sklearn.kernel_ridge import KernelRidge
     from sklearn.preprocessing import LabelEncoder
     from sklearn.metrics import mean_squared_error
-    from sklearn.cross_validation import LeaveOneLabelOut
+    try:
+        from sklearn.model_selection import (KFold, StratifiedKFold,
+                                             ShuffleSplit, LeaveOneLabelOut)
+        sklearn_version = 'new'
```

Comment:
why don't you use this: https://github.com/mne-tools/mne-python/blob/master/mne/utils.py#L739 ?